syntax = "proto2";

package NYT.NExecNodeTrackerClient.NProto;

import "yt_proto/yt/client/node_tracker_client/proto/node.proto";

////////////////////////////////////////////////////////////////////////////////

message TReqHeartbeat
{
    required int32 node_id = 1;

    required NNodeTrackerClient.NProto.TExecNodeStatistics statistics = 2;

    optional string job_proxy_build_version = 3;
}

message TRspHeartbeat
{
    optional bool disable_scheduler_jobs = 1;
}

////////////////////////////////////////////////////////////////////////////////
